nemo/collections/tts/data/data_utils.py

```diff
@@ -15,27 +15,10 @@
 import json
 import os
 from pathlib import Path
-from typing import List
 
 import numpy as np
 
 
-def read_manifest(manifest_path: Path) -> List[dict]:
-    """Read manifest file at the given path and convert it to a list of dictionary entries.
-    """
-    with open(manifest_path, "r", encoding="utf-8") as manifest_f:
-        entries = [json.loads(line) for line in manifest_f]
-    return entries
-
-
-def write_manifest(manifest_path: Path, entries: List[dict]) -> None:
-    """Convert input entries to JSON format and write them as a manifest at the given path.
-    """
-    output_lines = [f"{json.dumps(entry, ensure_ascii=False)}\n" for entry in entries]
```

Comment:
@rlangman, are you disabling `ensure_ascii` on purpose? it seems not necessary to do this. Please confirm.

Reply:
Yes, it is on purpose. By default ensure_ascii=True which means that all text is converted to ascii which messes up things like accented characters.

Reply:
ok, let me add such support then.